doc/PythonSupport.md

```diff
@@ -44,6 +44,13 @@ Here are several noteworthy modules we haven't supported yet:
  - `threading.Lock` - unsupported (yet)
 - `curses` - not supported
 - `tkinter` - not supported
+- `locale` - musl libc [has very limited locale support](https://wiki.musl-libc.org/open-issues.html). Python program running in Mystikos would suffer the same limitation, since Mystikos is using musl.
```

Comment:
```suggestion
- `locale` - musl libc [has very limited locale support](https://wiki.musl-libc.org/open-issues.html). Python programs running in Mystikos would suffer the same limitations, since Mystikos is using musl.
```